<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Segmentation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image : url('assets/img/back.png')</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="doPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Do watershed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="resetPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="exitPushButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="loadPushButton">
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="savePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <widget class="QMenu" name="menuAuthor">
     <property name="title">
      <string>Author</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionMohamed_Aimane_SKHAIRI"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuContact">
     <property name="title">
      <string>Contact</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionmedaimane_github_io"/>
     <addaction name="separator"/>
     <addaction name="actionskhairimedaimane_gmail_com"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionDescription"/>
    <addaction name="separator"/>
    <addaction name="menuAuthor"/>
    <addaction name="separator"/>
    <addaction name="menuContact"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDescription">
   <property name="text">
    <string>Description</string>
   </property>
  </action>
  <action name="actionMohamed_Aimane_SKHAIRI">
   <property name="text">
    <string>Mohamed Aimane SKHAIRI</string>
   </property>
  </action>
  <action name="actionmedaimane_github_io">
   <property name="text">
    <string>medaimane.github.io</string>
   </property>
  </action>
  <action name="actionskhairimedaimane_gmail_com">
   <property name="text">
    <string>skhairimedaimane@gmail.com</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
